build.gradle

```diff
@@ -18,10 +18,10 @@ plugins {
   id 'org.checkerframework' version '0.6.11' apply false
   id 'org.jetbrains.grammarkit' version '2021.2.2' apply false
   id 'org.jetbrains.intellij' version '1.6.0' apply false
-  // Minor Kotlin version is effectively enforced by the IntelliJ version we're building against,
-  // check https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library
-  // every time `intellijVersions.buildTarget` is bumped.
-  id 'org.jetbrains.kotlin.jvm' version '1.6.20' apply false
+  // As per https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library:
+  //   If a plugin supports multiple platform versions, it must (...) target the lowest bundled stdlib version
+  // which is 1.4.0 in our case (as this is the Kotlin version bundled into IntelliJ 2020.3).
+  id 'org.jetbrains.kotlin.jvm' version '1.4.0' apply false
```

Comment:
Downgrade
![not-stonks](https://user-images.githubusercontent.com/3383210/173386740-10c6bc90-6d35-40ff-b016-ea160b777b13.jpg)